### PR TITLE
SUMO-236061: Azure Append Blob Collection

### PR DESCRIPTION
**Pull request by priyansh-sumo**
## Purpose of this pull request

This pull request creates documentation for Append Blob Collection

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

https://sumologic.atlassian.net/browse/SUMO-236061

### CODE REVIEW

**Comment by kimsauce** [comment]
Checked in with apps team via Slack

**Comment by kimsauce** [comment]
@priyansh-sumo , just letting you know I've fixed the broken links and 301 redirects. For next time, [here are the steps to follow](https://help.sumologic.com/docs/contributing/remove-doc) when you rename or remove a doc.

**Comment by JV0812** [comment]
@kimsauce there was some path error with this PR and I fixed all the paths which were listed in the error. But still there is an error because of the cid-redirect files, we have added all the redirect link to the page. Can you please from your end once?

File: cid-redirects.json [hunk 1]
```diff
@@ -3872,6 +3872,9 @@
   "/docs/cse/automation-service/automation-service-bridge": "/docs/platform-services/automation-service/automation-service-bridge",
   "/docs/cse/automation-service/automation-service-integration-framework": "/docs/platform-services/automation-service/automation-service-integration-framework",
   "/docs/send-data/collect-from-other-data-sources/kubernetes": "/docs/send-data/kubernetes",
+  "/docs/send-data/collect-from-other-data-sources/azure-blob-storage": "/docs/send-data/collect-from-other-data-sources/azure-blob-storage/block-blob/collect-logs",
```
Comment (kimsauce) [inline cid-redirects.json:3875]: @JV0812 This is the line causing the error. It's redirecting from a file that already exists to another file that already exists. So you'd need to remove or correct this line.

File: blog-service/2019/12-31.md [hunk 2]
```diff
@@ -423,7 +423,7 @@ Update - The [Cylance App](/docs/integrations/security-threat-detection/cylance
...
 
-Update - The [Azure Network Watcher App](/docs/integrations/microsoft-azure/network-watcher) and [Azure Blob Storage App](/docs/send-data/collect-from-other-data-sources/azure-blob-storage) have an enhanced collection processes. The SumoLogic App for Azure Network Watcher leverages Network Security Group (NSG) flow logs to provide real-time visibility and analysis of your Azure Network. The Sumo Logic App for Azure Blob Storage is a Sumo integration that provides a event-based pipeline for shipping monitoring data from Azure Blob Storage to an HTTP source on Sumo Logic.
+Update - The [Azure Network Watcher App](/docs/integrations/microsoft-azure/network-watcher) and [Azure Blob Storage app](/docs/send-data/collect-from-other-data-sources/azure-blob-storage/block-blob/collect-logs) have an enhanced collection processes. The SumoLogic App for Azure Network Watcher leverages Network Security Group (NSG) flow logs to provide real-time visibility and analysis of your Azure Network. The Sumo Logic App for Azure Blob Storage is a Sumo integration that provides a event-based pipeline for shipping monitoring data from Azure Blob Storage to an HTTP source on Sumo Logic.
```
Comment (kimsauce) [inline blog-service/2019/12-31.md:426]:
```suggestion
Update - The Sumo Logic apps for [Azure Network Watcher](/docs/integrations/microsoft-azure/network-watcher) and [Azure Blob Storage](/docs/send-data/collect-from-other-data-sources/azure-blob-storage/block-blob/collect-logs) now have an enhanced collection processes. The Azure Network Watcher app leverages Network Security Group (NSG) flow logs to provide real-time visibility and analysis of your Azure Network. The Sumo Logic App for Azure Blob Storage provides an event-based pipeline for shipping and monitoring data from Azure Blob Storage to an HTTP source on Sumo Logic.
```

File: docs/send-data/collect-from-other-data-sources/azure-blob-storage/append-blob/collect-logs.md [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+In this step, you configure an HTTP source to receive logs from the Azure function.
+
+1. Select a hosted collector where you want to configure the HTTP source. If desired, create a new hosted collector, as described on [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
+1. Configure an HTTP source, as described on [HTTP Logs and Metrics Source](/docs/send-data/hosted-collectors/http-source/logs-metrics). Make a note of the URL for the source, you will need it in the next step.
+
+## Step 3. Configure Azure resources using ARM template
+
+In this step, you use a Sumo-provided Azure Resource Manager (ARM) template to create an Event Hub, three Azure functions, Service Bus Queue, and a Storage Account.
+
+1. Download the [appendblobreaderdeploy.json](https://raw.githubusercontent.com/SumoLogic/sumologic-azure-function/master/AppendBlobReader/src/appendblobreaderdeploy.json) ARM template.
```
Comment (jpipkin1) [inline docs/send-data/collect-from-other-data-sources/azure-blob-storage/append-blob/collect-logs.md:60]: The link to the appendblobreaderdeploy.json file is broken.

File: docs/send-data/collect-from-other-data-sources/azure-blob-storage/append-blob/index.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+For instructions, see [Collect Logs from Azure Blob Storage](/docs/send-data/collect-from-other-data-sources/azure-blob-storage/append-blob/collect-logs).
+
+## Azure Append Blob Limitations
+
+1. Limitations 1
+1. Limitations 2
+1. Limitations 3
+1. Limitations 4
+1. Limitations 5
```
Comment (jpipkin1) [inline docs/send-data/collect-from-other-data-sources/azure-blob-storage/append-blob/index.md:79]: Add the limitations, or remove this section.

File: docs/send-data/collect-from-other-data-sources/azure-blob-storage/troubleshoot-log-collection.md [hunk 5]
```diff
@@ -1,12 +1,13 @@
...
 sidebar_label: Troubleshooting
 description: Follow these steps to learn why log data is not flowing into Sumo from Azure Blob Storage.
 ---
+
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-If logs do not start flowing into Sumo Logic after you perform the [Collect Logs from Azure Blob Storage](collect-logs-azure-blob-storage.md) procedure, see the troubleshooting tips below.
+If logs do not start flowing into Sumo Logic after you perform the Collect Logs from Azure Blob Storage procedure, see the troubleshooting tips below.
 
 ## Error while deploying the ARM template
```
Comment (jpipkin1) [inline docs/send-data/collect-from-other-data-sources/azure-blob-storage/troubleshoot-log-collection.md:13]: In this section, the link for "Refer to this video" doesn't work.

**Review by jpipkin1 — APPROVED** [review]
Approved, @priyansh-sumo. I made a number of small changes, such as changing instances of "Sumo" to "Sumo Logic" and updating the index pages (see commit "[Updates from review](https://github.com/SumoLogic/sumologic-documentation/pull/3979/commits/17b9965bbb958ebc20ad249ee64d266b3b6db437)").

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
This still needs some work, @priyansh-sumo. See my comments.

File: docs/send-data/collect-from-other-data-sources/azure-blob-storage/append-blob/collect-logs.md (hunk 3 above)
Comment (himanshu219) [reply docs/send-data/collect-from-other-data-sources/azure-blob-storage/append-blob/collect-logs.md:61]: once the PR is merged to master (we are waiting for QE go ahead) then only the link becomes active

File: docs/send-data/collect-from-other-data-sources/azure-blob-storage/troubleshoot-log-collection.md (hunk 5 above)
Comment (himanshu219) [reply docs/send-data/collect-from-other-data-sources/azure-blob-storage/troubleshoot-log-collection.md:13]: removed the link for now, I couldn't find it.

**Review by jpipkin1 — APPROVED** [review]

**Pull request merged**